backend/src/main/java/de/neuefische/backend/BackendApplication.java

```diff
@@ -1,50 +1,18 @@
 package de.neuefische.backend;
 
-import de.neuefische.backend.model.User;
-import de.neuefische.backend.repository.MongoUserRepository;
-import de.neuefische.backend.service.MongoUserDetailsService;
+
 import org.apache.juli.logging.Log;
 import org.apache.juli.logging.LogFactory;
-import org.springframework.beans.factory.annotation.Autowired;
-import org.springframework.boot.CommandLineRunner;
 import org.springframework.boot.SpringApplication;
 import org.springframework.boot.autoconfigure.SpringBootApplication;
-import org.springframework.dao.DuplicateKeyException;
-import org.springframework.security.core.authority.SimpleGrantedAuthority;
-import org.springframework.security.crypto.password.PasswordEncoder;
-
-import java.util.List;
 
 @SpringBootApplication
-public class BackendApplication implements CommandLineRunner {
```

Comment:
sehr gut dass der CommandLineRunner raus ist, der lief bei mir dann immer auf Fehler im Produktivsystem, beziehungsweise in der CI